x-pack/solutions/observability/plugins/serverless_observability/public/navigation_tree.ts

```diff
@@ -511,6 +512,7 @@ export const createNavigationTree = ({
                     { link: 'management:anomaly_detection' },
                     { link: 'management:analytics' },
                     { link: 'management:trained_models' },
+                    { link: 'management:supplied_configurations' },
```

Comment:
if I remember correctly, a test explicitly checked for this, so I added it back. It was present in the old nav.

Reply:
Some things were inadvertently omitted from the serverless design tabs in the IA doc, this is possibly one of them.